wire-schema/src/jvmTest/kotlin/com/squareup/wire/schema/PruningRulesTest.kt

```diff
@@ -223,18 +238,20 @@ class PruningRulesTest {
         .exclude("a.IncludedType")
         .build()
     set.includes(ProtoMember.get("a.IncludedType#member"))
-    assertThat(set.unusedExcludes()).containsExactly("a.IncludedType")
+    assertThat(set.unusedExcludes()).containsExactly("a.*")
   }
 
   @Test
-  fun trackingUnusedPrecedence() {
```

Comment:
what happened to this?

Reply:
We cannot have conflicting rules anymore (this crashes). We already have tests when include/exclude are of different precision so I killed it.